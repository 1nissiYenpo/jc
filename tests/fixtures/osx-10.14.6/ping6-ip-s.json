{"source_ip": "::1", "destination_ip": "::1", "data_bytes": 1448, "pattern": null, "destination": "::1", "packets_transmitted": 3, "packets_received": 3, "packet_loss_percent": 0.0, "duplicates": 0, "round_trip_ms_min": 0.093, "round_trip_ms_avg": 0.135, "round_trip_ms_max": 0.161, "round_trip_ms_stddev": 0.03, "responses": [{"type": "reply", "bytes": 1408, "response_ip": "::1", "icmp_seq": 0, "ttl": 64, "time_ms": 0.093, "duplicate": false}, {"type": "reply", "bytes": 1408, "response_ip": "::1", "icmp_seq": 1, "ttl": 64, "time_ms": 0.161, "duplicate": false}, {"type": "reply", "bytes": 1408, "response_ip": "::1", "icmp_seq": 2, "ttl": 64, "time_ms": 0.152, "duplicate": false}]}
