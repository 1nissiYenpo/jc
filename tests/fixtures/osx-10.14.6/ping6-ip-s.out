PING6(1448=40+8+1400 bytes) ::1 --> ::1
1408 bytes from ::1, icmp_seq=0 hlim=64 time=0.093 ms
1408 bytes from ::1, icmp_seq=1 hlim=64 time=0.161 ms
1408 bytes from ::1, icmp_seq=2 hlim=64 time=0.152 ms

--- ::1 ping6 statistics ---
3 packets transmitted, 3 packets received, 0.0% packet loss
round-trip min/avg/max/std-dev = 0.093/0.135/0.161/0.030 ms
